linera-service/src/cli/main.rs

```diff
@@ -684,10 +696,87 @@ impl Runnable for Job {
                                 }
                                 RemoveValidator { public_key } => {
                                     if validators.remove(&public_key).is_none() {
-                                        warn!("Skipping removal of nonexistent validator");
+                                        warn!("Validator {public_key} does not exist; aborting.");
```

Comment:
I'd remove `aborting.` since that assumes execution stops.

if we actually interrupt execution, this should be an error, IMO